opentelemetry-sdk/src/metrics/internal/mod.rs

```diff
@@ -146,9 +146,8 @@ impl<AU: AtomicallyUpdate<T>, T: Number, O: Operation> ValueMap<AU, T, O> {
             let new_tracker = AU::new_atomic_tracker(self.buckets_count);
             O::update_tracker(&new_tracker, measurement, index);
             trackers.insert(STREAM_OVERFLOW_ATTRIBUTES.clone(), Arc::new(new_tracker));
-            global::handle_error(MetricsError::Other("Warning: Maximum data points for metric stream exceeded. Entry added to overflow. Subsequent overflows to same metric until next collect will not be logged.".into()));
-            otel_warn!( name: "ValueMap.measure",
-                message = "Maximum data points for metric stream exceeded. Entry added to overflow. Subsequent overflows to same metric until next collect will not be logged."
+            otel_error!( name: "ValueMap.Measure.Overflow",
```

Comment:
```suggestion
            otel_error!( name: "MetricCardinalityLimitReached",
```

This should be user actionable. We should include name of meter, instrument along with the configured limit, but those requires major refactoring.

Reply:
I think this can be a WARN. We can reserve ERROR for the case where entire telemetry is lost/dropped. In this case, we are not dropping entire measurement but folding into overflow.

Reply:
Have changed ERROR to WARN, and added the cardinality limit. With  TODO to add instrument name and metric name.